### PR TITLE
Sanitise item paths in loadouts 

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Check if an item path exist before trying to spawn it, and warns the user if it doesn't. 
Since an item path can be removed/changed in PRs, we have to sanitise them

## Why It's Good For The Game

Less runtime, user is warned that his loadout is a bit old

## Changelog
:cl:
code: Sanitise item paths in loadouts 
/:cl:

### CODE REVIEW

File: code/datums/loadout/item_representation/_item_representation.dm [hunk 1]
```diff
@@ -27,6 +27,9 @@
 /datum/item_representation/proc/instantiate_object(datum/loadout_seller/seller, master = null, mob/living/user)
 	if(seller && !bypass_vendor_check && !buy_item_in_vendor(item_type, seller, user))
 		return
+	if(!text2path(item_type))
+		to_chat(user, span_warning("[item_type] is not a correct path, it as probably changed or removed."))
```
Comment (TiviPlus) [inline code/datums/loadout/item_representation/_item_representation.dm:31]:
```suggestion
		to_chat(user, span_warning("[item_type] in your loadout is an invalid path, it as probably changed or removed."))
```
Also shouldnt it remove it from the loadout so it doesnt happen again

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/datums/loadout/item_representation/_item_representation.dm (hunk 1 above)
Comment (TiviPlus) [reply code/datums/loadout/item_representation/_item_representation.dm]: See comment

Comment (BraveMole) [reply code/datums/loadout/item_representation/_item_representation.dm]: way too much work for something that will rarely happen, and that you can fix by saving your loadout another time

Comment (BraveMole) [reply code/datums/loadout/item_representation/_item_representation.dm]: not as simple as a -= 

an incorrect item can be saved on anything. In a storage, on an armor, can be a gun attachment, etc.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**